frontend/src/pages/Chat/ChatRoom.tsx

```diff
@@ -0,0 +1,239 @@
+import { FormEvent, useContext, useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { LogoutIcon, SendIcon } from '../../assets/icons/icons';
+import ChatList from '../../components/Chat/ChatList';
+import ChatMessage from '../../components/Chat/ChatMessage';
+import Loading from '../../components/commons/Loading';
+import Modal from '../../components/commons/Modal/Modal';
+import CustomInput from '../../components/CustomInput';
+import PageHeader from '../../components/PageHeader/PageHeader';
+import { UserInfoContext } from '../../context/UserInfoContext';
+import { useModal } from '../../hooks/useModal';
+import useSocket from '../../hooks/useSocket';
+import { remote } from '../../lib/api';
+import { useNavigate, usePathParams } from '../../lib/Router';
+import colors from '../../styles/colors';
+import { textMedium, textSmall } from '../../styles/fonts';
+import { ChatRoomInfo } from '../../types/chat';
+
+function Chat() {
+  const [message, setMessage] = useState('');
+  const { chatId } = usePathParams();
+  const [chatRoomInfo, setChatRoomInfo] = useState<ChatRoomInfo>(null!);
```

Comment:
밑에서 chatRoomInfo?. 이런 식으로 옵셔널하게 접근하고 있으니까 not null assertion이 큰 의미가 없는 것 같아요!
초기에 chatRoomInfo가 없는건 확실하기에 undefined를 주고 쓰는게 더 나아보입니다!

```suggestion
  const [chatRoomInfo, setChatRoomInfo] = useState<ChatRoomInfo>();
```